Dockerfile-14.0

```diff
@@ -16,7 +16,6 @@ COPY ./install /tmp/install
 RUN set -x \
   && /tmp/install/pre-install.sh \
   && /tmp/install/tools.sh \
-  && /tmp/install/confd.sh \
```

Comment:
Don't forget to remove `install/confd.sh` too.
And to update the other Dockerfiles.